src/Deprecated.jl

```diff
@@ -1,13 +1,57 @@
 ##==============================================================================
 # deprecation staging area
 ##==============================================================================
+##==============================================================================
+## Deprecated in v0.9 Remove in the v0.10 cycle
+##==============================================================================
+
+# temporary promote with warning
+Base.promote_rule(::Type{DateTime}, ::Type{ZonedDateTime}) = DateTime
+function Base.convert(::Type{DateTime}, ts::ZonedDateTime)
+    @warn "DFG now uses ZonedDateTime, temporary promoting and converting to DateTime local time"
+    return DateTime(ts, Local)
+end
+
+@deprecate listSolvekeys(x...) listSolveKeys(x...)
```

Comment:
Keep these.

Reply:
I think we should make the promotion of DateTime to include timezone information permanent, not just a short term deprecated feature.

Reply:
The promotion is type piracy. So we can't leave it permanently.